assets/dora/man_route_opt_lb.yml

```diff
@@ -0,0 +1,24 @@
+---
+applications:
+- name: ((name))
```

Comment:
You don't need the `name` var. The `cf push APP_NAME...` command should overwrite whatever the app name is set to in the manifest.

Please replace this with a hardcoded name, e.g. "dora"